### PR TITLE
Review and update README

### PR DESCRIPTION
**Pull request by lread**
As a noob, I did not at first understand Polylith was separate from the Clojure poly tool and documented separately. I made this clearer in the README.
- changed the title to include "for Clojure"
- I used the term "conceptual" instead of "high-level" for Polylith gitbook docs.

I got very confused by the diagram.
It is very cool but I thought it deserved at least a high level description. I might have got it right, please do review.

Was not shy about using the word Polylith.
In a README, I felt it deserved to be repeated often.

Other changes were minor, like splitting up long sentences.

### CODE REVIEW

**Comment by lread** [comment]
@tengstrand, my first crack at the first file.

Lemme know what you think.
I'll not be offended at all if you want to make changes or don't like some of the changes I've made.

**Comment by tengstrand** [comment]
Hi!
You are right, the diagram looks cool, but is introduced too early. I suggest that we remove it as you suggest. I like the change to conceptual. The diagram shows up in Production systems (I moved it from Example systems, and the 318-branch is one commit ahead of yours).

I like your changes, and it will make it easier for people to start out with Polylith!

Great work!

File: readme.adoc [hunk 1]
```diff
@@ -8,46 +8,52 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
...
-This tool is made by developers for developers with the goal to maximize productivity and increase the quality of the systems we write.
-It supports your build pipeline but is not a build tool itself.
+Polylith is made by developers for developers.
+Our goal is to maximize productivity and increase the quality of the systems we create.
+Although Polylith is not a build tool, it integrates into your build pipeline.
 
-The Polylith concept can be implemented in any programming language, but this version of the Polylith tool targets
-https://clojure.org/[Clojure] which is a powerful and simple functional language for the
-https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiB88eLxansAhUyi8MKHd6jDPEQFjAAegQIBRAC&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJava_virtual_machine&usg=AOvVaw0YtnMyoG7GQIhUPeLulbfr[JVM].
+Polylith can be implemented in any programming language.
```
Comment (tengstrand) [inline readme.adoc]: I suggest that we skip this sentence entirely.

**Review by tengstrand — CHANGES_REQUESTED** [review]

File: readme.adoc (hunk 1 above)
Comment (lread) [reply readme.adoc]: Yeah, I think that make sense, it is now redundant.

**Comment by lread** [comment]
@tengstrand made those changes. Happy to tweak more if you have more ideas.

File: readme.adoc [hunk 2]
```diff
@@ -8,46 +8,36 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
...
-https://clojure.org/[Clojure] which is a powerful and simple functional language for the
-https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiB88eLxansAhUyi8MKHd6jDPEQFjAAegQIBRAC&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJava_virtual_machine&usg=AOvVaw0YtnMyoG7GQIhUPeLulbfr[JVM].
+Polylith, as a concept, is documented https://polylith.gitbook.io/polylith[here].
 
-Polylith introduces the architectural concept of “service level building blocks”, which can be combined like LEGO® bricks
-to build our services and systems. Polylith’s LEGO-like bricks are easy to reason about, test, refactor, and reuse.
-They allow us to work with all our code in one place for maximum productivity, using a single
-https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop[REPL].
+The documentation you are reading now describes the Polylith tool built for https://clojure.org/[Clojure].
+Clojure is a powerful and simple functional language for the https://en.wikipedia.org/wiki/Java_virtual_machine[JVM].
```
Comment (tengstrand) [inline readme.adoc]: I think it flows better if we merge these two sentences "...describes the Polylith tool built for [Clojure](https://clojure.org/), which is a powerful ...
To get rid of the repetition "Clojure. Clojure".

File: readme.adoc [hunk 3]
```diff
@@ -8,46 +8,36 @@ https://cljdoc.org/d/polylith/clj-poly/0.2.18-SNAPSHOT[image:https://badgen.net/
...
 
 Enjoy the ride!
 
 == Sponsor Polylith
 
 Please https://github.com/sponsors/polyfy[sponsor] the important work of Polylith! ❤️
 
-Iron, Bronze, Silver, Gold, and Platinum sponsors will also be listed on the
-https://polylith.gitbook.io/polylith[first page] of the high-level documentation.
+Iron, Bronze, Silver, Gold, and Platinum sponsors are also listed and on the https://polylith.gitbook.io/polylith[front page of the Polylith docs].
```
Comment (tengstrand) [inline readme.adoc]: Maybe it's enough to put the link in "front page": https://polylith.gitbook.io/polylith[front page] of the Polylith docs.

**Review by tengstrand — CHANGES_REQUESTED** [review]

File: readme.adoc (hunk 2 above)
Comment (lread) [reply readme.adoc]: I've personally got no problem with the 2 Clojures. 
I think smaller sentences are easier to digest.

Comment (lread) [reply readme.adoc]: But, of course, your call.

File: readme.adoc (hunk 3 above)
Comment (lread) [reply readme.adoc]: Sure, can do.

File: readme.adoc (hunk 2 above)
Comment (lread) [reply readme.adoc]: Meh, I'll change it as suggested.

Comment (lread) [reply readme.adoc]: Done!

Comment (tengstrand) [reply readme.adoc]: Okay, thanks!

**Pull request merged**